www/scss/partials/_chantbook.scss

```diff
@@ -7,6 +7,10 @@
   color: #3c3c3c;
 }
 
+#chantbook-list {
+  margin-bottom: 85px;
```

Comment:
I think this should be handled on iOS and Android separately since this adds too much extra space below the chant list on iOS devices due to the bottom toolbar. I like how it looks on Android, so making it look the same on iOS would be nice.